domain/miningmanager/miningmanager_test.go

```diff
@@ -225,8 +225,10 @@ func TestHandleNewBlockTransactions(t *testing.T) {
 		if err != nil {
 			t.Fatalf("HandleNewBlockTransactions: %v", err)
 		}
-		if len(miningManager.AllTransactions()) != 0 {
-			blockIDs := domainBlocksToBlockIds(miningManager.AllTransactions())
+		mempoolTransactions, _ = miningManager.AllTransactions(true, false)
+		if len(mempoolTransactions) != 0 {
+			mempoolTransactions, _ = miningManager.AllTransactions(true, false)
```

Comment:
Why have you added this line?

Reply:
at least this line and the orphan retrival.